.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish package to the Maven Central Repository
+
+on:
+  release:
+    types: [created]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: fission-sonatype
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+          gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
```

Comment:
Just curious why one is coming from a secret and rest are just keys?

Reply:
> Just curious why one is coming from a secret and rest are just keys?

Not sure, I followed the standard from https://github.com/actions/setup-java#publishing-using-apache-maven

But I think that's because that particular field is not needed as environment variable